libs/entitlement/src/main/java/org/elasticsearch/entitlement/runtime/policy/PolicyManager.java

```diff
@@ -538,11 +538,13 @@ private void notEntitled(String message, Class<?> callerClass) {
         var exception = new NotEntitledException(message);
         // Don't emit a log for muted classes, e.g. classes containing self tests
         if (mutedClasses.contains(callerClass) == false) {
+            var moduleName = callerClass.getModule().getName();
+            var notEntitledLogger = LogManager.getLogger(PolicyManager.class + "." + ((moduleName == null) ? "ALL_UNNAMED" : moduleName));
```

Comment:
nit: we use `ALL-UNNAMED` (with a dash) in the policy files

Reply:
Why the module name and not the component name? (plugin name, "server" or "apm-agent"?)
Wouldn't we have clash for ALL-UNNAMED from different plugins if we need to adjust log level for one non-modular plugin?

Reply:
The component name is also reasonable, but the finer grained we can make this the less changing this logging could affect other log messages.

Reply:
whoops

Reply:
Having both as agreed is the best 👍  Thanks!